.golangci.yml

```diff
@@ -6,9 +6,8 @@ linters-settings:
   gofmt:
     simplify: true
   goimports:
-    local-prefixes: github.com/mattermost/mattermost-plugin-profanity-filter
+    local-prefixes: github.com/mattermost/mattermost-starter-template
```

Comment:
Is this change intentional?  I think the plugin package should be here as well.

Reply:
Nope. Fixing.